File: app/src/main/java/com/justnik/mirea_viewbinding_lesson/ExampleActivity.java
package com.justnik.mirea_viewbinding_lesson;

import androidx.appcompat.app.AppCompatActivity;

import android.app.Activity;
import android.os.Bundle;

import com.justnik.mirea_viewbinding_lesson.databinding.ActivityExampleBinding;

public class ExampleActivity extends AppCompatActivity {

    private ActivityExampleBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityExampleBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        launchExampleFragment();
    }

    private void launchExampleFragment() {
        ExampleFragment fragment = ExampleFragment.newInstance();

        getSupportFragmentManager().beginTransaction()
                .replace(R.id.example_container, fragment)
                .commit();
    }
}

File: app/src/main/java/com/justnik/mirea_viewbinding_lesson/ExampleActivity.kt
package com.justnik.mirea_viewbinding_lesson

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import by.kirich1409.viewbindingdelegate.viewBinding
import com.justnik.mirea_viewbinding_lesson.databinding.ActivityExampleBinding

class ExampleActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_example)
        launchExampleFragment()
    }

    private fun launchExampleFragment(){
        val fragment = ExampleFragment.newInstance()

        supportFragmentManager.beginTransaction()
            .replace(R.id.example_container, fragment)
            .commit()
    }
}